draft/2023-08-16-this-week-in-rust.md

```diff
@@ -187,33 +187,25 @@ Rusty Events between 2023-08-16 - 2023-09-13 🦀
 
 ### North America
 
-* 2023-08-10 | Mountain View, CA, US | [Mountain View Rust Meetup](https://www.meetup.com/mv-rust-meetup/)
-    * [**Rust Meetup at Hacker Dojo**](https://www.meetup.com/mv-rust-meetup/events/294911475/)
-* 2023-08-10 | Lehi, UT, US | [Utah Rust](https://www.meetup.com/utah-rust/)
-    * [**Building a simplified JVM in Rust**](https://www.meetup.com/utah-rust/events/294972766/)
-* 2023-08-15 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
-    * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/vwljctyfclbtb)
-* 2023-08-15 | Seattle, WA, US | [Seattle Rust User Group Meetup](https://www.meetup.com/seattle-rust-user-group/)
-    * [**Seattle Rust User Group - August Meetup**](https://www.meetup.com/seattle-rust-user-group/events/294804636/)
 * 2023-08-16 | Cambridge, MA, US | [Boston Rust Meetup](https://www.meetup.com/bostonrust/)
     * [**Alewife Rust Lunch**](https://www.meetup.com/bostonrust/events/294910746/)
-* 2023-08-16 | Copenhagen, DK | [Copenhagen Rust Community](https://www.meetup.com/copenhagen-rust-community/)
-    * [**Rust metup #39 sponsored by Fermyon**](https://www.meetup.com/copenhagen-rust-community/events/294806394)
 * 2023-08-17 | Nashville, TN, US | [Seattle Rust User Group Meetup](https://www.meetup.com/music-city-rust-developers/)
     * [**Rust goes where it pleases. Rust on the web and embedded**](https://www.meetup.com/music-city-rust-developers/events/294805470/)
 * 2023-08-23 | Austin, TX, US | [Rust ATX](https://www.meetup.com/rust-atx/)
     * [**Rust Lunch - Fareground**](https://www.meetup.com/rust-atx/events/295008514)
 * 2023-08-24 | Mountain View, CA, US | [Mountain View Rust Meetup](https://www.meetup.com/mv-rust-meetup/)
     * [**Rust Meetup at Hacker Dojo**](https://www.meetup.com/mv-rust-meetup/events/295107743/)
+* 2023-08-30 | Copenhagen, DK | [Copenhagen Rust Community](https://www.meetup.com/copenhagen-rust-community/)
+    * [**Rust metup #39 sponsored by Fermyon**](https://www.meetup.com/copenhagen-rust-community/events/294806394)
 * 2023-09-06 | Bellevue, WA, US | [The Linux Foundation](https://www.linuxfoundation.org/)
     * [**Rust Global**](https://events.linuxfoundation.org/rust-global/)
+* 2023-09-12 - 2023-09-15 | Albuquerque, NM, US  + Virtual | [RustConf](https://rustconf.com/)
+    * [**RustConf 2023**](https://rustconf.com/)
 
 ### Oceania
 
-* 2023-08-09 | Perth, WA, AU | [Rust Perth](https://www.linkedin.com/groups/7439562/)
```

Comment:
We have organised another one for next month - I've just now emailed it to community-team@rust-lang.org
```markdown
* 2023-09-14 | Perth, WA, AU | [Rust Perth](https://www.linkedin.com/groups/7439562/)
    * [**September Meetup**](https://www.tickettailor.com/events/perthrustusergroup/984771)
```
no worries if it is too late to be included in this issue.

Reply:
Thanks for the submission! We will make sure to include this in the next issue. Please don't hesitate to make a PR for submissions as well -- this is the most expeditious way to get a timely review.